Frends.FTP.UploadFiles/Frends.FTP.UploadFiles.Tests/Lib/UploadFilesTestBase.cs

```diff
@@ -21,20 +21,20 @@ protected bool DummyFileExists(string fileName, string dir = null)
         return File.Exists(Path.Combine(dir ?? Dir, fileName));
     }
 
-    protected string CreateDummyDir()
+    protected static string CreateDummyDir()
     {
         var dir = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
         Directory.CreateDirectory(dir);
         return dir;
     }
-    
-    [SetUp]
+
+    [TestInitialize]
     public void SetUp()
     {
         Dir = CreateDummyDir();
     }
 
-    [TearDown]
+    [TestCleanup]
     public void TearDown()
     {
         Directory.Delete(Dir, true);
```

Comment:
There should be OneTimeTearDown which could clean all the test data in folder DockerVolumes\data after tests.

Reply:
Decided to do the cleanup inside TearDown()s so there's no need to use NUnit and save some valuable rows.